src/codemagic/tools/app_store_connect.py

```diff
@@ -127,9 +133,9 @@ def _get_resource(self, resource_id, resource_manager, should_print):
         self.printer.print_resource(resource, should_print)
         return resource
 
-    def _list_resources(self, resource_filter, resource_manager, should_print):
+    def _list_resources(self, resource_filter, resource_manager, should_print, **kwargs):
```

Comment:
right, ordering will be deleted anyway. But instead of kwargs will put `limit` to be able to sort `builds` by `-uploadedDate` and will `limit: 1` get the build with latest build number as Fastlane does